app/helpers/detailed-date.js

```diff
@@ -0,0 +1,15 @@
+import { helper } from '@ember/component/helper';
+import lightFormat from 'date-fns/lightFormat';
+
+export default helper(function detailedDate([datetime]/*, hash*/) {
+  //If not a date (e.g. date is undefined) return "" for printing on screen.
+  if (!(datetime instanceof Date)) return "";
```

Comment:
I know it's merged already, but this check seems redundant since the try catch handles it already? And the comment above seems redundant as well since it's literally the transcript of what you think when you read the actual line of code :smile:.

Reply:
I thought the intention with the try...catch was to print an error in case the actual dates did not make sense, but we don't want an error when the date is just undefined, because that can happen in a normal situation. I wanted error messages only when really necessary to not cause confusion later.
Yes, the comment is pretty useless like this, sorry for that, but at least it means this check was put there for a reason.